index.html

```diff
@@ -164,6 +164,27 @@ <h3 class="skill">Skills</h3>
                 </div>
             </div>
         </section>
+        <footer class="contact">
+            <h2>I'm always <br> interested in hearing about new projects, 
+                so if you'd like to chat please get in</h2>
+            <div class="ip-type">
+                <input class="Name" type="text" placeholder="    Full Name" maxlength="30" required>
+                <input class="email" type="email" placeholder="   Email address" required>
+                <textarea name="msgt1" class="t1" cols="30" rows="10" maxlength="500" placeholder="  Enter text here"></textarea>
+                <button class="get" type="submit">Get in touch</button>
+            </div>
```

Comment:
- [ ] Kindly put the contact form inside a form tag
- [ ] Kidly make sure that your form submit to [formspree service](https://formspree.io/html)